doc/getting-started/overview_dev.rst

```diff
@@ -208,67 +206,23 @@ and :ref:`overview_dev_build`.
 |icon_host| Step 4: Generate a Scenario Configuration File and Launch Scripts
 *****************************************************************************
 
-The :ref:`acrn_configurator_tool` ``acrn_configurator.py`` enables you to
-configure your ACRN hypervisor and VMs via a web-based user interface on your
-development computer. Using the tool, you define your scenario settings and save
-them to a scenario configuration file. For scenarios with post-launched User
-VMs, you must also configure and generate launch scripts.
+The :ref:`acrn_configurator_tool`, found in the ACRN hypervisor source code,
```

Comment:
I believe we'll actually have a Debian package for the 3.0 release that will be used to install the configurator.
```suggestion
The :ref:`acrn_configurator_tool`
```